modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Configure PrivateLink and Private Service Connect in the Cloud UI
+:description: Set up AWS PrivateLink and GCP Private Service Connect in the Redpanda Cloud UI.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+NOTE: This guide is for configuring AWS PrivateLink and GCP Private Service Connect using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] or xref:deploy:deployment-option/cloud/gcp-private-service-connect.adoc[Configure GCP Private Service Connect for BYOC] if you want to set up these services using the API.
+
+The Redpanda AWS PrivateLink and GCP Private Service Connect services provide secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink and Private Service Connect do not go through the public internet because these connections are treated as their own private AWS and GCP services. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using the endpoint services if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink and Private Service Connect allow overlapping CIDR ranges in VPC networks.
+* PrivateLink and Private Service Connect do not limit the number of VPC connections.
+* You control which GCP projects or AWS resources are allowed to connect to the endpoint services.
```

Comment:
@paulzhang97 want to confirm that this is also the case for PrivateLink (I took this item from the [PSC doc](https://deploy-preview-516--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/gcp-private-service-connect/) )

Reply:
Specifically, AWS manages control at the principal level.

Reply:
Yes. I would like to change 2nd bullet at line 14 to `PrivateLink and Private Service Connect do not limit the number of connections connected via Private Link or Private Service Connect.`

In 3rd bullet, change `AWS resources` to `AWS principals`.